oauth2_provider/settings.py

```diff
@@ -128,13 +134,13 @@ class OAuth2ProviderSettings:
 
     def __init__(self, user_settings=None, defaults=None, import_strings=None, mandatory=None):
         self.user_settings = user_settings or {}
-        self.defaults = defaults or {}
-        self.import_strings = import_strings or ()
+        self.defaults = defaults or DEFAULTS
```

Comment:
why you decided to pass DEFAULTS if we're not passing anything? This looks legit to me, tho I am wondering why it was {} and not DEFAULTS before

Reply:
I noticed that the library settings are very similar to the django-rest-framework settings.
My changes are inspired from [there](https://github.com/encode/django-rest-framework/blob/master/rest_framework/settings.py#L202)

This will allow [tests](https://github.com/jazzband/django-oauth-toolkit/pull/904/files#diff-f20069a79c955c52519f37001105df187ba738b9a4dd49af010c723ff810200cR17) to cover configuration errors. Previously, there were no tests for settings at all.

Reply:
makes sense, thanks!